### ISSUE TITLE
lower_acl_than_parent not working for private(set)

### ISSUE THREAD

**Issue by PrzemyslawCholewaDev** [realm/SwiftLint#2709]
This line triggers lower_acl_than_parent warning:
fileprivate struct Foo { private(set) var bar }

It shouldn't, right? :)

**Comment by akashivskyy** [comment]
It shouldn't... and it doesn't. 🤔

```swift
// file.swift

fileprivate struct Foo {
  private(set) var bar: Int
}
```

```yml
# swiftlint.yml

whitelist_rules:
  - lower_acl_than_parent
```

```none
$ swiftlint lint --config swiftlint.yml --path file.swift

Loading configuration from 'swiftlint.yml'
Linting Swift files at paths file.swift
Linting 'file.swift' (1/1)
Done linting! Found 0 violations, 0 serious in 1 file.
```

**Comment by jpsim** [comment]
Thanks for checking @akashivskyy
